src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml

```diff
@@ -81,17 +84,135 @@
                     <RowDefinition Height="*" />
                 </Grid.RowDefinitions>
 
-                <TabControl TabStripPlacement="Left" Background="{StaticResource PreferencesWindowBackgroundColor}">
+                <TabControl TabStripPlacement="Left" 
+                            Background="{StaticResource PreferencesWindowBackgroundColor}">
+                    
+                    <!--General Tab-->
                     <TabItem Header="{x:Static p:Resources.PreferencesViewGeneralTab}"
                              Style="{StaticResource LeftTab}">
-                        <Label Content="General Content goes here..." 
-                               Foreground="{StaticResource PreferencesWindowFontColor}"/>
+
+                        <!--This Grid contains controls for selecting the Language-->
+                        <Grid x:Name="GeneralTab" 
+                              Margin="12 0 0 0"
+                              HorizontalAlignment="Left">
+                            <Grid.RowDefinitions>
+                                <RowDefinition Height="*" />
+                                <RowDefinition Height="*" />
+                                <RowDefinition Height="*" />
+                                <RowDefinition Height="*" />
+                                <RowDefinition Height="*" />
+                            </Grid.RowDefinitions>
+                            <Grid Grid.Row="0">
+                                <StackPanel Orientation="Vertical">
+                                    <Label  Content="Language" 
```

Comment:
I would prefer no hard coded resource string in XML because they will be missing localization. Can you look for if we already have similar res string for you to use?

Reply:
@QilongTang 
I searched in the resources files but we don't have a similar entry, then I added a new resource entry, see the changes in the commit:
https://github.com/DynamoDS/Dynamo/pull/11542/commits/9035f3d2b7de82a15cb6098d2a56a00c1b06836d